### PR TITLE
#951: Global time zone setting for timestamps and dated

### PR DESCRIPTION
**Pull request by benedeki**
* Definition of the two new settings for Standardization and their usage in TypeParser
* Increased project version

### CODE REVIEW

**Comment by benedeki** [comment]
**For Release Notes**
Definition of the two new settings for Standardization (`application.conf`)
`defaultTimestampTimeZone` - time zone used for timestamp fields standardization if none specified in the schema
`defaultDateTimeZone` - time zone used for date fields standardization if none specified in the schema; recommended not to define

**Comment by benedeki** [comment]
_Hotfix notes:_
I am aware that the ` private val defaultTimeZone: Option[String]` is defined both in `DateParser` and `TimestampParser` while it could be in their common ancestor.
Also the way the setting is done, it's hard to test other then with an E2E test.

File: standardization/src/main/resources/application.conf.template [hunk 1]
```diff
@@ -17,3 +17,8 @@ standardized.hdfs.path="/tmp/conformance-output/standardized-{0}-{1}-{2}-{3}"
 
 #system-wide time zone
 timezone="UTC"
+#time zone used for timestamp fields standardization if none specified in the schema
+#defaultTimestampTimeZone
```
Comment (Zejnilovic) [inline standardization/src/main/resources/application.conf.template]: Does this equal nothing?

Comment (Zejnilovic) [reply standardization/src/main/resources/application.conf.template]: Default is UTC, set by the code in TimeZoneNormalizer, right?

Comment (benedeki) [reply standardization/src/main/resources/application.conf.template]: > 
> 
> Does this equal nothing?

Sorry. Added a better example.

Comment (benedeki) [reply standardization/src/main/resources/application.conf.template]: > 
> 
> Default is UTC, set by the code in TimeZoneNormalizer, right?

Default is the value of `timezone` setting. Which in the current state of code should not deviate from _UTC_.

**Pull request merged**